include/seqan/index/index_fm.h

```diff
@@ -205,6 +205,16 @@ class Index<TText, FMIndex<TSpec, TConfig> >
     typename Fibre<Index, FibreLF>::Type            lf;
     typename Fibre<Index, FibreSA>::Type            sa;
 
+    /*!
+     * @fn FMIndex::Index
+     * @brief Constructor
+     *
+     * @signature Index::Index();
+     * @signature Index::Index(text);
+     *
+     * @param[in] text The text to be indexed.
+     */
```

Comment:
> If so, we should drop a node in the description of the descriptor.

Yes, please document that the all the talk about SATree and FibreSA is misleading, because the FM-Index in the current implementation does not provide a suffx trie interface, but a prefix trie interface. 

> Maybe he can explain why he doesn't use a ModifiedString instead. Performance reasons?

Yes, I recommond using `reverse()` because it moves the transformation to time-of-indexing, instead of time-of-access.